python/python/glide/async_commands/core.py

```diff
@@ -6002,6 +6003,115 @@ async def watch(self, keys: List[TEncodable]) -> TOK:
             await self._execute_command(RequestType.Watch, keys),
         )
 
+    async def xclaim(
+        self,
+        key: TEncodable,
+        group: TEncodable,
+        consumer: TEncodable,
+        min_idle_time_ms: int,
+        ids: List[TEncodable],
+        options: Optional[StreamClaimOptions] = None,
+    ) -> Mapping[bytes, List[List[bytes]]]:
+        """
+        Changes the ownership of a pending message.
+
+        See https://valkey.io/commands/xclaim for more details.
+
+        Note:
+            When in cluster mode, the command may route to multiple nodes when `keys` map to different hash slots.
```

Comment:
please remove this.  There is only a single `key` for this command, so this message doesn't apply ;)

Reply:
addressed